src/components/Result.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+
+export default class Result extends Component {
+  constructor(props){
+    super(props);
+  }
+
+  render(){
+    let result = this.props.result;
+    return (
+      <tr>
+        <td>{result.title}</td>
+        <td>{result.description.split("\n").map( (line) => {
+          return (
+            <span>
+              {line}
+              <br />
+            </span>
+          )
+        })}</td>
+        <td>{result.assignees.join(", ")}</td>
```

Comment:
@ajvb This code Went from CrypTask to Backchannel and now to LeapChat lol. There are no assignees anywhere except for CrypTask